config/initializers/assets.rb

```diff
@@ -5,13 +5,8 @@
 
 # Add additional assets to the asset load path
 Rails.application.config.assets.paths.push(
-  'node_modules',
```

Comment:
From a developer experience perspective, I actually preferred having `node_modules` as a load path and requiring fully-qualified paths to specific dependencies. It still works in Propshaft; however, Propshaft will copy all files from load paths into `public/assets`, which essentially would mean copying over the entirety of `node_modules`.